runner/bsp/worker/src/mill/bsp/worker/MillJvmBuildServer.scala

```diff
@@ -41,9 +41,12 @@ private trait MillJvmBuildServer extends JvmBuildServer { this: MillBuildServer
       targetIds: collection.Seq[BuildTargetIdentifier],
       agg: java.util.List[JvmEnvironmentItem] => V
   )(implicit name: sourcecode.Name): CompletableFuture[V] = {
+    val logger = createLogger()
     handlerTasks(
       targetIds = _ => targetIds,
-      tasks = { case m: RunModuleApi => m.bspJvmRunTestEnvironment }
+      tasks = { case m: RunModuleApi => m.bspJvmRunTestEnvironment },
+      requestDescription = "jvmRunTestEnv",
```

Comment:
Do we need custom names for these? We already take the `sourcecode.Name` of each endpoint method, which I would have thought would give us enough information about what endpoint each log message is comming from

Reply:
They're used to build messages that end up as dialog messages when using VSCode (lower right corner). `requestDescription` is meant to be used to build more user-friendly messages. I tweaked it a bit in the latest push.